### ISSUE TITLE
[Ops] Support for Avgpool3d op (ttnn.Avgpool3d)

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-metal#15102]
**Description**

- Avgpool3d op support is needed for Alexnet model compilation.

**Comment by fahmedchanTT** [comment]
@pavlejosipovic Do we support or plan to support AvgPool3d?

**Comment by pavlejosipovic** [comment]
Not ATM.

**Comment by pavlejosipovic** [comment]
@kamalrajkannan78  why was this closes as completed?

**Comment by kamalrajkannan78** [comment]
> [@kamalrajkannan78](https://github.com/kamalrajkannan78) why was this closes as completed?

I’m currently doing a TT-Metal ticket cleanup for my items.
It appears there is no plan to support this at the moment, so I closed the issue.
Would you prefer that I reopen it and label it as `do-not-close`?

**Comment by pavlejosipovic** [comment]
Yes I would prefer that, but in general I don't like closing this as "done",  if you want to close better option is "not planned" as it might give an impression like the feature was implemented.

**Comment by kamalrajkannan78** [comment]
As Request got deprioritized, I am closing it now
